images/tests/Dockerfile.rhel

```diff
@@ -12,7 +12,7 @@ COPY --from=builder /go/src/github.com/openshift/origin/zz_generated.manifests/*
 
 RUN PACKAGES="git gzip util-linux" && \
     if [ $HOSTTYPE = x86_64 ]; then PACKAGES="$PACKAGES python3-cinderclient"; fi && \
-    if [ $HOSTTYPE = x86_64 ]; then PACKAGES="$PACKAGES rt-tests rteval"; fi && \
+    if [ $HOSTTYPE = x86_64 ]; then PACKAGES="$PACKAGES realtime-tests rteval"; fi && \
```

Comment:
The CI reports `No match for argument: realtime-tests`. So this rpm is not available?

Reply:
Wrong base image is being used, it's rhel8, and should be rhel9